projects/common-consumption/src/lib/content-details/player-toc/player-toc.component.scss

```diff
@@ -0,0 +1,270 @@
+      display:flex;
+      align-items: center;
+      justify-content: space-between;
+      cursor: pointer;box-shadow: 0 4px 7px 0 rgba(0,0,0,0.1);
+      &--bg-white {
+        background:var(--white);
+      }
+      h4 {
+        font-size:1em;
+        color:var(--primary-400);margin: 0;
+        padding: 0px;
+        font-weight:bold;
+        width:100%;
+      }
+    }
+    
+    .sbtoc-filter {
+      height:calc(var(--base-block-space) * 8);
+      padding:var(--base-block-space) calc(var(--base-block-space) * 2);
+      box-shadow: 0px 2px 10px -5px rgba(0, 0, 0, 0.25);display: inline-flex;
+      align-items: center;
+      width:100%;overflow: hidden;
+      overflow-x: auto;
+      border-bottom:1px solid #e0e0e0;
+      // &::-webkit-scrollbar {
+      //   height:0px;
+      // }
+      &--bg-gray{
+      background:var(--gray-0);
+      }
+      &__grid {
+        display: inline-flex;
+        align-items: center;
+        margin: 0px;
+        padding: 0px;
+      }
+      &__grid-item{
+        border-radius: calc(var(--base-block-space) * 3);
+        padding:  calc(var(--base-block-space));
+        margin: 0px;
+        margin-right:var(--base-block-space);
+        display: inline-flex;
+        align-items: center;background:var(--white);
+        font-size:0.8em;white-space: nowrap;
+        cursor: pointer;min-width: 48px;justify-content: center;
+        .icon-svg{
+          margin-right:4px;
+        }
+      }
+      &__grid-item--active, &__grid-item:hover {
+        background:var(--primary-400);
+        color:var(--white);
+        .icon-svg{
+          svg {
+            fill:var(--white);
+          } 
+        }
+      }
+    }
+    
+    
+    .sbaccordion {
+      width:100%;
+      &--toc{
+      }
+      &__panel {
+        margin-bottom:8px;
+        box-shadow: 0 4px 7px 0 rgba(0,0,0,0.1);
+      }
+      &__panel-header {
+        background:var(--white);
+        padding:var(--base-block-space) calc(var(--base-block-space) * 2);
+        display:flex;
+        align-items: center;
+        justify-content: space-between;
+        cursor: pointer;
+        height:calc(var(--base-block-space) * 6);
+      }
+      &__panel-header__title{
+        font-size:14px;
+        font-weight:bold;
+        color:var(--primary-400);
+        margin:0px;padding:0px;width:100%;
+      }
+      &__panel-content{
+        border-top:1px solid #E0E0E0;
+        background:var(--gray-0);
+        min-height: auto;
+        overflow:hidden;
+        overflow-y:auto;
+        max-height:300px;
+      }
+      .sbchapter {
+        border-bottom:1px solid #ddd;
+        padding:calc(var(--base-block-space) * 2);
+        h4{
+          font-size:10px;
+          color:#333;
+          margin-left:16px;
+        }
+        &__item{
+          background:var(--white);
+          border:1px solid #ddd;
+          box-shadow: 0 3px 4px 0 rgba(2,79,157,0.1);

```

Comment:
Box shadow Colors Seems to be in 5 variant colors in this component alone. Need to review with design team for the need of common color for shadows. Having one variant is fine